lib/tasks/allrecipe-category-tags.rake

```diff
@@ -0,0 +1,44 @@
+task :category_tags => :environment do 
+  get_category_info("Initial Level", "http://allrecipes.com/recipes/main.aspx?vm=l")
+end
+
+#run infinitly as long as there is a sub_category
+def get_category_info(main_category_name, main_category_url)
+  require 'open-uri'
+  require 'nokogiri'
+
+  @tries = 0
+  begin
+    puts main_category_name
+    puts main_category_url
+    browser = open(main_category_url).read
+    html_doc = Nokogiri::HTML(browser)
+
+    sub_category_elements = html_doc.css('a#hlSubNavItem')
+    if sub_category_elements.any?
+      sub_category_elements.each do |sub_category_element|
+        #prevent inconsistent href links
+        sub_category_url = sub_category_element.attr("href") + "&vm=l"
```

Comment:
vm=1 changes to listing method from grid to list. 
if i dont change to list, i couldn't get the author information